examples/calibration/plot_compare_calibration.py

```diff
@@ -86,17 +92,19 @@ def predict_proba(self, X):
 
 from sklearn.calibration import CalibrationDisplay
 from sklearn.ensemble import RandomForestClassifier
-from sklearn.linear_model import LogisticRegression
+from sklearn.linear_model import LogisticRegressionCV
 from sklearn.naive_bayes import GaussianNB
 
 # Create classifiers
```

Comment:
```suggestion
# Create classifiers: here we tune the main hyper-parameter automatically for
# the logistic regression model because its efficient and concise to do so.
# Ideally, we should run an hyper-parameter search for all the models but we
# don't do it here for the sake of keeping the example code concise and fast
# to execute.
```